4.analyze_data/notebooks/UMAP/scripts/1.visualize-UMAP-plate3.r

```diff
@@ -209,6 +207,10 @@ print(merged_plot)
 ggsave(output_file, merged_plot, dpi = 500, height = 12, width = 12)
 
 
+install.packages("ggExtra")
+
+library(ggExtra)
```

Comment:
would it be too much of a hassle to add to the conda environment? Also, libraries should be loaded in the beginning of the notebooks

Reply:
Not a hassle at all! Added!